Tests/mocks/interfaces/IFocus.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
please add this to the ThunderInterfaces repo

gerrit uses ThunderInterfaces repo and unit tests use ThunderInterfaces repo

this file should not be here